<launch>
	<node pkg="multi_cam"	type="multi_cam.py"	name="multi_cam_0" output="screen">
		<param name="/camera_ID" type="int" value="0"/>
		<remap from="/multi_cam" to="multi_cam0" />
	</node>
	<node pkg="multi_cam"	type="multi_cam.py"	name="multi_cam_1" output="screen">
		<param name="/camera_ID" type="int" value="1"/>
		<remap from="/multi_cam" to="multi_cam1" />
	</node>
	<node pkg="multi_cam"	type="multi_cam.py"	name="multi_cam_2" output="screen">
		<param name="/camera_ID" type="int" value="2"/>
		<remap from="/multi_cam" to="multi_cam2" />
	</node>
	<node pkg="multi_cam"	type="multi_cam.py"	name="multi_cam_3" output="screen">
		<param name="/camera_ID" type="int" value="3"/>
		<remap from="/multi_cam" to="multi_cam3" />
	</node>

</launch>
